components/ToggleButton.js

```diff
@@ -23,27 +23,24 @@ const breakpoint = 768,
   teal = '#23485f'
 
 const Button = styled.button`
+  @apply px-4 py-2 border-0 text-white;
   appearance: none;
-  color: white;
-  padding: 15px;
-  border: none;
 
   background: ${props => props.isActive ? teal : 'black' };
```

Comment:
```suggestion
  ${props => props.isActive ? 'bg-teal' : 'bg-black' };
```

Reply:
FYI you probably know this, but just in case... 

I see you added the teal color in `tailwind.config.js`, we can use it in [backgrounds too](https://tailwindcss.com/docs/background-color#customizing-your-theme).
![image](https://user-images.githubusercontent.com/25883086/226728541-a56bae1d-936d-4c0d-a4c2-b781a70d21c2.png)

Reply:
I think this would need to be something like

```suggestion
  @apply ${props => props.isActive ? 'bg-teal' : 'bg-black' };
```

RIght?  And I wonder how that would work, like if `@apply` works with one of these interpolated prop functions.  I don't think it would.

On Amp, @isaazgarcia, in cases like this, would you just conditionally toggle classes on and off from `className`?  Like:

```jsx
<Button className={ clsx(
  'px-4 py-2 border-0 text-white appearance-none',
  isActive ? 'bg-teal' : 'bg-black'
)} />
```

Reply:
Oh yeah, the `@apply` needs to be there. 

> would you just conditionally toggle classes on and off from className?

Yes! exactly like that!